app/views/pages/new_project.html.erb

```diff
@@ -0,0 +1,172 @@
+              <div class="mt-4 space-y-4">
+                <div class="flex items-center">
+                  <input id="person_category_bride" name="person_category" type="radio" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300">
+                  <label for="person_category_bride" class="ml-3 block text-sm font-medium text-gray-700">
+                    <%= t('bride') %>
+                  </label>
+                </div>
+                <div class="flex items-center">
+                  <input id="person_category_groom" name="person_category" type="radio" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300">
+                  <label for="person_category_groom" class="ml-3 block text-sm font-medium text-gray-700">
+                    <%= t('groom') %>
+                  </label>
+                </div>
+                <div class="flex items-center">
+                  <input id="person_category_celebrant" name="person_category" type="radio" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300">
+                  <label for="person_category_celebrant" class="ml-3 block text-sm font-medium text-gray-700">
+                    <%= t('celebrant') %>
+                  </label>
+                </div>
+              </div>
+            </fieldset><!-- Person category block end -->
+            <div class="grid grid-cols-3 gap-6"><!-- Email address block start -->
+              <div class="col-span-3 sm:col-span-2">
+                <label for="email_address" class="block text-sm font-medium text-gray-700 required">
+                  <%= t('pages-new_project-email_address-label') %><!-- Your email address is -->
+                </label>
+                <input type="text" name="email_address" id="email_address" class="focus:ring-indigo-500 focus:border-indigo-500 flex-1 block w-full border-gray-300 rounded py-2 px-3 sm:text-sm leading-tight" placeholder="me@example.com">
+                <p class="mt-2 text-sm text-gray-500 italic">
+                  <%= t('pages-new_project-email_address-description') %> <!-- Your email address will never be shared without your permission. Wedding booklet projects you create will be associated with your email address.  -->
+                </p>
+              </div>
+            </div><!-- Email address block end -->
+            <div class="grid grid-cols-3 gap-6"><!-- Marriage date block start -->
+              <div class="col-span-3 sm:col-span-2">
+                <label for="marriage_date" class="block text-sm font-medium text-gray-700 required">
+                  <%= t('pages-new_project-marriage_date-label') %><!-- The wedding will take place on: -->
+                </label>
+                <input type="datetime-local" name="marriage_date" id="marriage_date" class="focus:ring-indigo-500 focus:border-indigo-500 flex-1 block w-full border-gray-300 rounded py-2 px-3 sm:text-sm leading-tight">
```

Comment:
Could there be a placeholder here? It's unclear what kind of time/date format would be best.

Reply:
I think pretty much all browsers now have a calendar picker? That should pretty much take care of getting the format right, because it will change between locales anyways and browsers handle that with the calendar picker.

Reply:
For example my Chrome browser is set to Italian, so it renders the the datetime input like this:
![datetime input](https://user-images.githubusercontent.com/231298/123777929-f0d1a380-d8d0-11eb-8223-9bf77a7e3ec7.jpg)

gg = giorno = day
mm = mese = month
aaaa = anno = year

I'm guessing in an English browser it will show up something like this:
`mm/dd/yyyy`
Month - Day - Year